miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultBoxService.java

```diff
@@ -544,4 +544,15 @@ public void throwIfBoxPositionIsFilled(Boxable boxable) throws IOException {
     }
   }
 
+  @Override
+  public void beforeDelete(Box object) throws IOException {
+    object.getBoxPositions().clear();
+    save(object);
+  }
+
+  @Override
+  public void authorizeDeletion(Box object) throws IOException {
+    authorizationManager.throwIfNonAdminOrMatchingOwner(object.getCreator());
```

Comment:
Please add the standard note to the delete dialog in `list_box.js` to go with this ("Note: a Box may only be deleted by its creator or an admin.")

Reply:
Done